Documentation/ceph-upgrade.md

```diff
@@ -51,23 +52,23 @@ With this upgrade guide, there are a few notes to consider:
 
 Unless otherwise noted due to extenuating requirements, upgrades from one patch release of Rook to
 another are as simple as updating the common resources and the image of the Rook operator. For
-example, when Rook v1.5.3 is released, the process of updating from v1.5.0 is as simple as running
+example, when Rook v1.6.1 is released, the process of updating from v1.6.0 is as simple as running
 the following:
 
-First get the latest common resources manifests that contain the latest changes for Rook v1.5.
+First get the latest common resources manifests that contain the latest changes for Rook v1.6.
 ```sh
-git clone --single-branch --branch v1.5.3 https://github.com/rook/rook.git
+git clone --single-branch --depth=1 --branch v1.6.1 https://github.com/rook/rook.git
```

Comment:
`depth=1` looks good, we should apply it in the docs as well where we use the `single-branch` flag, but we could do it in a separate PR to avoid the merge conflicts with the backport. Those lines have all been modified in the release branch.